crates/solidity/Cargo.toml

```diff
@@ -41,6 +41,18 @@ inkwell = { workspace = true }
 revive-common = { workspace = true }
 revive-llvm-context = { workspace = true }
 
-
 [target.'cfg(target_env = "musl")'.dependencies]
 mimalloc = { version = "*", default-features = false }
+
+[target.'cfg(target_os = "emscripten")'.dependencies]
+libc = { workspace = true }
+inkwell = { workspace = true, features = ["target-riscv", "llvm18-0-no-llvm-linking"]}
+
+[features]
+default = []
+parallel = ["rayon"]
+
+#TODO: Use pthreads -C target-feature=+atomics,+bulk-memory -Clink-arg=-pthread in compilation and enable `parallel` feature
+# Enable parallel_native by default only for non-emscripten targets
+[target.'cfg(not(target_os = "emscripten"))'.features]
+default = ["parallel_native"]
```

Comment:
Why do we even add this feature if we don't use it for emscripten? Couldn't we then just sticked to `rayon`? Why do we have two different parallel features now?

Reply:
yes, there should be just one parallel feature. I am going to have emscripten with and without parallel execution. I already have PoC.